File: Aula2/Idade/Idade.cpp
#include <iostream>

int main()
{
    int ano_atual, ano_nascimento, idade_atual;
    std::string nome;
    
    std::cin >> ano_atual;
    std::cin >> nome;
    std::cin >> ano_nascimento;
    
    idade_atual = ano_atual - ano_nascimento;
    
    std::cout << nome << ", voce completa " << idade_atual << " anos de idade neste ano." << std::endl;
    return 0;
}

File: Aula1/Idade/Idade.java
import java.util.Scanner;

public class Idade{
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int anoAtual = sc.nextInt();
        String nome = sc.next();
        int anoNascimento = sc.nextInt();
        int idade = anoAtual - anoNascimento;
        
        System.out.println(nome + ", voce completa " + idade + " anos de idade neste ano.");
    }
}